crypto/src/proof_of_work.rs

```diff
@@ -95,11 +95,11 @@ impl ProofOfWork {
 // `check_proof_of_work(&received_raw_data[4..60], target)`
 pub fn check_proof_of_work(data: &[u8], target: f64) -> PowResult {
     let target_number = make_target(target);
-    check_proof_of_work_inner(data, &target_number)
+    check_proof_of_work_inner(data, &target_number).map_err(|_| ())
```

Comment:
@akoptelov @vlad9486 
this PowResult looks little bit hacky :D

can we do it here better with From trait:
`.map_err(PowResult::from)` instead of `.map_err(|_| ())` ?

Reply:
@akoptelov 
there is some compilation error,
please run `cargo bench` locally and see

Reply:
> @akoptelov @vlad9486
> this PowResult looks little bit hacky :D
> 
> can we do it here better with From trait:
> `.map_err(PowResult::from)` instead of `.map_err(|_| ())` ?

Hmm... we just don't need that at all.

And below we need to convert `Blake2bError` explicitly since `PowResult` is an alias for `Result<(), ()>`.